lib/ts/apidefs.h.in

```diff
@@ -710,7 +710,9 @@ typedef enum {
   TS_CONFIG_HTTP_CONNECT_ATTEMPTS_MAX_RETRIES_DEAD_SERVER,
   TS_CONFIG_HTTP_CONNECT_ATTEMPTS_RR_RETRIES,
   TS_CONFIG_HTTP_CONNECT_ATTEMPTS_TIMEOUT,
+  TS_CONFIG_HTTP_CONNECT_ATTEMPTS_TIMEOUT_MS,
```

Comment:
These changes in the enum make this incompatible with previous versions. Not a problem per se but does make back porting more perilous.